doc/tools/coqrst/coqdomain.py

```diff
@@ -891,18 +896,22 @@ def add_coq_output_1(self, repl, node):
         pairs = []
 
         if options['restart']:
-            repl.sendone("Restart.")
+            repl.sendone('Restart.')
```

Comment:
This seems a bit gratuitous

Reply:
Yes, I am a bit obsessed with consistency 😆